app/src/stylesheets/scss/dialogs/send-token.scss

```diff
@@ -177,6 +177,25 @@ md-dialog {
                         width: 210px;
                     }
                 }
+                .sk-send-ether-box__.select-token-container{
+                    position: absolute;
+                    right: 0;
+                    width: 28%;
+                    md-input-container{
```

Comment:
no need to create style for md-select .. there is already done restyling of **md-select** component.. 
use ```md-select.sk-select``` [check this](https://github.com/SelfKeyFoundation/Identity-Wallet/blob/sql-lite-impl/app/src/stylesheets/scss/components/md-select.scss)